src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,365 @@
-# Booting Kubernetes on Amazon Elastic Compute
```

Comment:
So most of this is copypasta from https://github.com/kubernetes/kops/blob/master/docs/aws.md as an FYI